test/images/README.md

```diff
@@ -0,0 +1,25 @@
+This image is the base image for Kubeflow Pipelines [Prow job](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubeflow/kubeflow-postsubmits.yaml#L245). Including presubmit and postsubmit jobs.
+
+## How to build the image
+To build this Docker image, run the following Docker command from the directory containing the image's files:
+
+``` 
+docker build -t test-worker . 
```

Comment:
QQ: why don't we tag it as `gcr.io/ml-pipeline-test/test-worker:${WHATEVER_TAG_VERSION}` then we can save the effort of retagging in the next step.